featuretools/tests/computational_backend/test_calculate_feature_matrix.py

```diff
@@ -917,13 +918,12 @@ def test_dask_persisted_es(es, capsys):
 
 class TestCreateClientAndCluster(object):
     def test_user_cluster_as_string(self, monkeypatch):
-        monkeypatch.setitem(create_client_and_cluster.__globals__,
-                            'LocalCluster',
-                            mock_cluster)
-        monkeypatch.setitem(create_client_and_cluster.__globals__,
-                            'Client',
-                            MockClient)
+        from featuretools.computational_backends import utils
```

Comment:
I think we can just import this once at the top of the file.  It's ok to import even if distributed isn't installed since the distributed import happens inside the `get_client_cluster` function.  The monkeypatch changes are fixed between tests so we don't need to reimport.

Reply:
Fixed.